fastddsspy_participants/src/cpp/model/DataStreamer.cpp

```diff
@@ -117,12 +118,13 @@ void DataStreamer::add_data(
         }
         else
         {
-            if (!(activated_topic_ == topic))
+            if (!(activated_topic_.matches(topic)))
```

Comment:
Please fix warning above ("but schema not is available" -> "but schema is not available")